userspace/chisel/chisel_viewinfo.cpp

```diff
@@ -179,7 +179,7 @@ void chisel_view_info::set_sorting_col()
 		throw sinsp_exception("view format error: more than one sorting column");
 	}
 
-	if((int64_t)m_sortingcol < 0)
+	if(m_sortingcol < 0)
```

Comment:
Here, the type of `m_sortingcol` was unsigned and the warning was lamenting that the value could never be negative. The type has been converted to signed.